src/unix/linux-core.c

```diff
@@ -107,6 +109,26 @@ int uv__platform_loop_init(uv_loop_t* loop) {
 }
 
 
+int uv__platform_loop_init(uv_loop_t* loop) {
+  return uv__epoll_init(loop, 1);
+}
+
+
+int uv__io_fork(uv_loop_t* loop) {
+  int err;
+  uv__close(loop->backend_fd);
+  if (loop->inotify_fd != -1) {
+    uv__io_stop(loop, &loop->inotify_read_watcher, POLLIN);
+    uv__close(loop->inotify_fd);
+    loop->inotify_fd = -1;
+    err = uv__init_inotify(loop);
+    if (err)
+      return err;
+  }
+
+  return uv__epoll_init(loop, 0);
```

Comment:
Closing the backend fd and running `uv__platform_loop_init` would result in the backend fd changing. But the backend fd is exposed to the application via `uv_backend_fd` and as such may be stored and used in other places; it may even have been customized. So I don't think we can do that.